ekwipunek.py

```diff
@@ -0,0 +1,823 @@
+
+    @property
+    def strzaly(self):
+        return self._strzaly
+
+    @property
+    def ogien(self):
+        return self._ogien
+
+    @property
+    def magia(self):
+        return self._magia
+
+
+class Magia(Przedmiot):
+    """
+    Klasa Magii.
+
+    :param nazwa: nazwa przedmiotu
+    :param wartosc: nazwa przedmiotu wyrażana w sztukach złota
+    :param mana: koszt many
+    :param efekt: efekt to zamiennik pola uzycie, jak rowniez bonusy ->
+        jego obecnosc informuje iz przedmiot mozna uzyc, a sam efekt informuje
+        o tym co przedmiot robi
+    :param krag: wymagany krąg magii, znajduje się on wtedy gdy mamy do
+        czynienia z runą i jest to jedyny sposób na ich rozróżnienie
+    """
+
+    def __init__(
+        self,
+        nazwa: str,
+        wartosc: int,
+        mana: int,
+        efekt: dict[str, int],
+        krag: int = None,
+    ):
+        super().__init__(nazwa, wartosc, efekt)
+        self._mana = mana
+        self._efekt = efekt
+        self._krag = krag
+
+    @property
+    def mana(self):
+        return self._mana
+
+    @property
+    def efekt(self):
+        return self._efekt
+
+    @property
+    def krag(self):
+        return self._krag
+
+
+class Zwoj(Magia):
+    """
+    Klasa Zwój dziedzicząca z klasy Magia.
+    """
+    pass
+
+
+class Runa(Magia):
+    """
+    Klasa Runa dziedzicząca z klasy Magia.
+    """
+    pass
+
+
+class Pismo(Przedmiot):
+    """
+    Klasa Pism - wyjątkowa klasa, ponieważ pisma nie można "ubrać". Użycie danej pozycji powoduje
+    wyprintowanie treści do konsoli.
+
+    :param nazwa: nazwa przedmiotu
+    :param wartosc: nazwa przedmiotu wyrażana w sztukach złota
+    :param tresc: treść danego pisma
+    :param efekt: efekt to zamiennik pola uzycie, jak rowniez bonusy ->
+        jego obecnosc informuje iz przedmiot mozna uzyc, a sam efekt informuje
+        o tym co przedmiot robi
+    """
+
+
+
+    def __init__(
+        self, nazwa: str, wartosc: int, tresc: str, efekt: dict[str, int] | None | str
+    ):
+        super().__init__(nazwa, wartosc, efekt)
+        self._tresc = tresc
+
+    @property
+    def tresc(self):
+        return self._tresc
+
+    def zastosowanie(self):
+        print(self._tresc)
+
+
+class Jedzenie(Przedmiot):
+    """
+    Klasa Jedzenie - Kolejna klasa, której nie da się ubrać. Użycie danej pozycji powoduje
+    jej zjedzenie, przez co pozycja zmniejsza swoją ilość w ekwipunku o jeden, lub znika z
+    ekwipunku.
+
+    :param nazwa: nazwa przedmiotu
+    :param wartosc: nazwa przedmiotu wyrażana w sztukach złota
+    :param efekt: efekt to zamiennik pola uzycie, jak rowniez bonusy ->
+        jego obecnosc informuje iz przedmiot mozna uzyc, a sam efekt informuje
+        o tym co przedmiot robi
+    """
+
+    def __init__(
+        self,
+        nazwa: str,
+        wartosc: int,
+        efekt: dict[str, int] | list,
+    ):
+        super().__init__(nazwa, wartosc, efekt)
+
+    def zastosowanie(self):
```

Comment:
to możesz usunąć

Reply:
zamiast zastosowanie -> w przypadku metody uzyj bedzie to wyswietlenie efektu